qcut/resources/default-skills/organize-project/Skill.md

```
@@ -0,0 +1,144 @@
+---
+name: organize-project
+description: Organize files and folders into QCut's standard project structure. Use when setting up a new project, cleaning up files, or when the user asks to organize their workspace.
+---
+
+# QCut Project Organization Skill
+
+Organize files and folders according to QCut's standard project structure.
+
+## Standard QCut Project Structure
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)